src/main/java/at/hannibal2/skyhanni/config/features/event/hoppity/HoppityCallWarningConfig.java

```diff
@@ -0,0 +1,49 @@
+package at.hannibal2.skyhanni.config.features.event.hoppity;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.utils.OSUtils;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorButton;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorColour;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorKeybind;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorText;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import io.github.notenoughupdates.moulconfig.observer.Property;
+import org.lwjgl.input.Keyboard;
+
+public class HoppityCallWarningConfig {
+
+    @Expose
+    @ConfigOption(name = "Hoppity Call Warning", desc = "Warn when hoppity is calling you.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
```

Comment:
Please make it default enabled. Imagine someone in support complaining "because this dumb feature was opt-out, I accidentally got a unique rabbit, so unfair"